### PR TITLE
Remove TODO comment for the verification of committed seals

### PR DESCRIPTION
**Pull request by Kourin1996**
# Description

This PR removes TODO comment from verifyCommitedFields because the behavior is correct
The changes as follows:
* Remove TODO comment from verifyCommitedFields in consensus/ibft/sign.go because this check is correct
* Add some comments
* Fix unfriendly error message

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [ ] I have updated the README and other relevant documents (guides...)
- [x] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯

**Review by lazartravica — APPROVED** [review]
LGTM

File: consensus/ibft/sign.go [hunk 1]
```diff
@@ -88,12 +88,12 @@ func writeCommittedSeals(h *types.Header, seals [][]byte) (*types.Header, error)
...
 
 	if len(seals) == 0 {
-		return nil, fmt.Errorf("bad")
+		return nil, fmt.Errorf("empty committed seals")
 	}
 
 	for _, seal := range seals {
 		if len(seal) != IstanbulExtraSeal {
-			return nil, fmt.Errorf("bad")
+			return nil, fmt.Errorf("invalid committed seal")
```
Comment (Vuksan) [inline consensus/ibft/sign.go]: Just a bit more descriptive 🙂 
```suggestion
			return nil, fmt.Errorf("invalid committed seal length")
```

**Review by lazartravica — APPROVED** [review]
LGTM

**Review by Vuksan — APPROVED** [review]
Looks good.

**Pull request merged**